src/sync.ts

```diff
@@ -1254,11 +1254,12 @@ export class SyncApi {
 
             const inviter = room.currentState.getStateEvents(EventType.RoomMember, client.getUserId()!)?.getSender();
 
-            if (client.isCryptoEnabled()) {
-                const parkedHistory = await client.crypto!.cryptoStore.takeParkedSharedHistory(room.roomId);
```

Comment:
For context, `client.isCryptoEnabled` checks if `client.cryptoBackend` is defined, rather than `client.crypto`. 

`CryptoBackend` doesn't have a `cryptoStore`. Eventually we can look at implementing `takeParkedSharedHistory` on `CryptoBackend`, but not today.